jobs/ci-kubernetes-node-kubelet-dockerized.sh

```diff
@@ -23,10 +23,11 @@ set -o xtrace
 readonly testinfra="$(dirname "${0}")/.."
 
 export KUBE_VERIFY_GIT_BRANCH='master'
```

Comment:
don't need this, though you might need something similar assuming we get different node image versions.